ta/arch/arm/link.mk

```diff
@@ -103,12 +105,23 @@ ifeq ($(CFG_ENCRYPT_TA),y)
 crypt-args$(user-ta-uuid) := --enc-key $(TA_ENC_KEY)
 cmd-echo$(user-ta-uuid) := SIGNENC
 endif
+ifeq ($(TA_SIGN_METHOD),stitch)
```

Comment:
Should we set a default value to TA_SIGN_METHOD, in mk/config.mk (i.e. `TA_SIGN_METHOD=sign-enc`)?

Reply:
Not necessary. The script defaults to signing.

Reply:
`sign_enc.py` assumes sign-enc is the default command, so I'd rather keep it as is for simplicity.

Reply:
It makes the makefiles implementation not that obvious i think. TA_SIGN_METHOD referred only at 2 places, only 1 value found, alternative is its default non-existence empty value.
But i'm fine as-is if you think it's easier to maintain that way.